src/styles/components/dropdown-menus/styles.less

```diff
@@ -98,6 +98,7 @@
     // Overrides default Canvas styles.
     float: initial;
     left: initial;
+    text-rendering: optimizeLegibility;
```

Comment:
Sometimes I was seeing a text flicker in the dropdown panel as well. This can be hard to reproduce, but easiest to reproduce by opening the user account dropdown while on the Nodes tab.

I suspect this has something to do with the fact that the Dropdown component animates opacity and changes z-index at the same time, and inside of it's animation's keyframe declaration.